content/get-started/quickstart/contributing-to-projects.md

```diff
@@ -95,6 +95,34 @@ gh repo fork REPOSITORY --clone=true
 
 {% enddesktop %}
 
+## Creating a branch to work on
+
+Before making changes to the project, you should create a new branch to commit them to. By keeping changes in their own branch, you follow [GitHub Flow](/get-started/quickstart/github-flow#following-github-flow) and ensure that it will be easier to contribute to the same project again in the future.
+
+{% webui %}
+
+```shell
+git branch branch-name
+git checkout branch-name
+```
+
+{% endwebui %}
+
+{% cli %}
+
+```shell
+git branch branch-name
+git checkout branch-name
+```
```

Comment:
We generally capitalize placeholders, so suggest changing to the following. 

````suggestion
```shell
git branch BRANCH-NAME
git checkout BRANCH-NAME
```

{% endwebui %}

{% cli %}

```shell
git branch BRANCH-NAME
git checkout BRANCH-NAME
```
````